### PR TITLE
Change userId from an int to an unsigned int

### PR DESCRIPTION
**Pull request by silverware13**

### CODE REVIEW

**Comment by philectron** [comment]
The type of `userId` ought to be `bigint`. According to [this MySQL 8.0 reference](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html), the maximum value of unsigned `int` is `4294967295` (which is compatible with 11-digit integers).

**Review by philectron — CHANGES_REQUESTED** [review]
The type of `userId` ought to be `bigint`. According to [this MySQL 8.0 reference](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html), the maximum value of unsigned `int` is `4294967295` (which is not compatible with 11-digit integers).

File: other/test_db.sql [hunk 1]
```diff
@@ -376,19 +375,19 @@ ALTER TABLE `Course`
...
 ALTER TABLE `PlanReview`
-  MODIFY `reviewId` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=55;
+  MODIFY `reviewId` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=54;
 
 --
 -- AUTO_INCREMENT for table `User`
 --
 ALTER TABLE `User`
-  MODIFY `userId` int(11) UNSIGNED NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=18;
+  MODIFY `userId` bigint(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=18;
```
Comment (philectron) [inline other/test_db.sql]: Oh snap, I forgot to check this. Should we not have auto increment on `userId`? The only time where we create a new User in the DB is when a user logs in via ONID but doesn't have their `osuuid` in the DB, and thus we create one with `userId` field set manually.

**Review by philectron — COMMENTED** [review]
Auto-increment on `userId` or not?

File: other/test_db.sql (hunk 1 above)
Comment (silverware13) [reply other/test_db.sql]: Yeah I can fix it. Also I am unsure why the unsigned vanished after being loaded into the DB so I will take another look.

**Review by philectron — APPROVED** [review]

**Pull request merged**